htdocs/langs/fr_FR/companies.lang

```diff
@@ -163,13 +163,13 @@ ProfId4CL=-
 ProfId5CL=-
 ProfId6CL=-
 ProfId1CM=Identifiant. prof. 1 (Registre de Commerce)
-ProfId2CM=Identifiant. prof. 2 (numéro de contribuable)
+ProfId2CM=Identifiant. prof. 2 (N° de contribuable)
```

Comment:
To change the translation of a language, you must edit the translation on
https://explore.transifex.com/dolibarr-association/dolibarr/
PR are useless because lang file in git are erased by the value in transifex.

Reply:
Please @eldy can any one validet my need on transifex